src/module/identifier/model.cpp

```
@@ -0,0 +1,367 @@
+        auto pad_y         = 0.0F;
+
+        {
+            adapt_scaling = std::min(static_cast<float>(1. * cols / origin_mat.cols),
+                                     static_cast<float>(1. * rows / origin_mat.rows));
+
+            const auto scaled_w = static_cast<int>(1. * origin_mat.cols * adapt_scaling);
+            const auto scaled_h = static_cast<int>(1. * origin_mat.rows * adapt_scaling);
+
+            // Calculate padding for center alignment
+            const auto pad_w_int = (cols - scaled_w) / 2;
+            const auto pad_h_int = (rows - scaled_h) / 2;
```

Cppcheck (2.19.0)

[style] The function 'printable' is never used.

(unusedFunction)